skimage/feature/texture.py

```diff
@@ -291,3 +293,130 @@ def local_binary_pattern(image, P, R, method='default'):
     image = np.ascontiguousarray(image, dtype=np.double)
     output = _local_binary_pattern(image, P, R, methods[method.lower()])
     return output
+
+def multiblock_local_binary_pattern(int_image, x, y, width, height):
+    """Multi-block local binary pattern.
+
+    MB-LBP is an extension of LBP that can be computed on many
+    scales in a constant time using integral image. It consists of
+    9 equal-sized rectangles. Sum of pixels' intensity values
```

Comment:
Replace `many` with `any`, as this function is only doing it for any 1 scale.
And I am not sure why we should maintain OpenCV API compatibility ( as you said on your blog ). No matter what API we choose, the arguments during the evaluation phase can be adjusted. I bring this up because, to me, mentioning the size of the outer rectangle seems more intuitive, because the artifact you are trying to quantize, is approximately the size of the bigger rectangle.